onset_maproom/calc.py

```diff
@@ -400,58 +389,52 @@ def seasonal_onset_date(
     dry_spell_search,
     time_coord="T",
 ):
-    """ Computes yearly seasonal onset dates from daily soil moisture data.
-
-        Function reproducing Ingrid onsetDate function 
-        '<http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate>`_
-        
-        Combining a function that groups data by season and a function that searches for an onset date
-
-        Computes yearly dates by utilizing groupby function to group data by season 
-        and onset_date function to calculate cessation date for each season.
-
-        Parameters
-        ----------
-        daily_rain : DataArray
-                    Array of daily rainfall values.
-        search_start_day : int
-                    The day part (1-31) of the date to start scanning for onset date.
-        search_start_month : int
-                    The month part (1-12) of the date to start scanning for onset date.
-        search_days : int
-                    Number of days from search start date to scan for onset date.
-        wet_thresh : int
-                    Rainfall threshold to determine wet day.
-        wet_spell_length : int
-                    Length in days of running window when 'wet_thresh' is to be met to define a wet spell.
-        wet_spell_thresh : int
-                    Threshold of rainfall to be reached during 'wet_spell_length' 
-                    window to define a wet spell. 
-        min_wet_days : int
-                    Minimum number of wet days in 'wet_spell_length' window when it rained at or above 
-                    'wet_spell_thresh' to be considered a wet spell.
-        dry_spell_length : int
-                    Length in days of dry spell that would invalidate a wet spell as onset date 
-                    if found within the 'dry_spell_search' days following the 'wet_spell_length'-day window 
-                    that met the wet spell criteria.      
-        dry_spell_search : int
-                    Length in days to search for a 'dry_spell_length'-day dry spell after a wet spell 
-                    is found that would invalidate the wet spell as onset date. 
-        time_coord : str, optional
-                    Time grid in 'soil_moisture' (the default is "T", which implies
-                    naming convention of time grid as "T" in 'soil_moisture')
-        Returns
-        -------
-        seasonal_cess_date : Dataset
-                    Dataset containing days since search start date as timedelta,
-                    and cessation date as datetime for each year in soil moisture DataArray.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
+    """Use daily soil moisture data to compute yearly seasonal onset dates from daily rainfall.
```

Comment:
Use daily soil moisture data to
that part is incorrect. There is no soil moisture data involved here.